source/linux/Foundational_Components_Edge_AI.rst

```diff
@@ -4,10 +4,22 @@
 Edge AI
 *******
 
-.. ifconfig:: CONFIG_part_family in ('AM62AX_family')
-
    The SDK provides software and tools to let the users effectively balance deep learning performance with system power and cost on Texas Instrument’s processors for edge AI applications. We offer a practical embedded inference solution for next-generation vehicles, smart cameras, edge AI boxes, and autonomous machines and robots. In addition to general purpose micro processors, AM62Ax has integrated micro controllers, DSP, and accelerators for neural network, image, vision, and multimedia processing. With a few simple steps one can run high performance computer vision and deep learning demos using a live camera and display.
 
    The SDK also enables an interplay of multiple open-source components such as GStreamer, OpenVx, OpenCV and deep learning runtime such as TFLite, ONNX and Neo-AI DLR. The reference applications showcase perception based examples such as image classification, object detection and semantic segmentation in both Python and C++ variants. The SDK supports edit-build-debug cycles directly on the target and also on PC to cross compile and build the applications.
 
-   For more information on edge AI software stack, refer `Edge AI Documentation <https://software-dl.ti.com/jacinto7/esd/processor-sdk-linux-edgeai/AM62AX/08_06_00/exports/docs/common/sdk_overview.html>`_
+.. ifconfig:: CONFIG_part_family in ('AM62AX_family')
+
+   For more information on edge AI software stack, refer `Edge AI Documentation <https://software-dl.ti.com/processor-sdk-linux/esd/AM62AX/10_00_00/exports/edgeai-docs/common/sdk_overview.html#>`_
```

Comment:
Will this be updated to point to 10_01_00 edgeai docs later?

Reply:
Yes